addons/website_event/tests/test_event_internals.py

```diff
@@ -30,18 +46,10 @@ def test_menu_management_frontend(self):
             'website_menu': True,
             'community_menu': False,
         })
-        self.assertTrue(event.website_menu)
-        self._assert_website_menus(event)
+        self._assert_website_menus(event, ['Introduction', 'Location', 'Register'], menus_out=['Community'])
 
         introduction_menu = event.menu_id.child_id.filtered(lambda menu: menu.name == 'Introduction')
         introduction_menu.unlink()
```

Comment:
Could be one-lined?
Also I'm not sure I fully understand the purpose of this test.
Is it to simulate the deletion of a menu through the website editor and check that the menu is correctly removed within the event structure?

Would be nice to add a small comment.

Reply:
Indeed this is the purpose, I add a comment.